tabpy/tabpy_server/common/default.conf

```diff
@@ -1,10 +1,10 @@
 [TabPy]
-TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
-TABPY_PORT = 9004
-TABPY_STATE_PATH = ./tabpy-server/tabpy_server
+# TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
+# TABPY_PORT = 9004
+# TABPY_STATE_PATH = ./tabpy/tabpy_server
 
 # Where static pages live
-TABPY_STATIC_PATH = ./tabpy-server/tabpy_server/static
+# TABPY_STATIC_PATH = ./tabpy/tabpy_server/static
 
 # For how to configure TabPy authentication read
 # docs/server-config.md.
```

Comment:
this should be updated to 
[Authentication](#authentication) 
instead of server-config.md

Reply:
This file is just plaintext and links won't work in it. Authentication explained in the server-config.md.